bootstrap.sh

```diff
@@ -93,7 +93,7 @@ if [ -n "$build" ] ; then
 	docker build -t "$DEV_IMG" --target=jdk docker-compose
 
 	# 2. run the maven build
-	mkdir -p /tmp/home
+	mkdir -p /src/tmp/home
```

Comment:
I would rather just put `mkdir -p tmp/home` here. The /src directory is inside the container (and mapped to the current directory with `-v "$PWD:/src"`), there is no point to create it outside the container (neither to force the user to put his sources into /src)